### DISCUSSION TITLE
The response content is getting lost on queued listener

### DISCUSSION THREAD

**Discussion by duongtdvn** [spatie/laravel-webhook-server#68] [Q&A]
So, I am listening for the `WebhookCallSucceededEvent` and `WebhookCallFailedEvent` events to save the log. While everything work as expected with the following listener class like so:
```php
<?php

namespace App\Listeners;

class StoreWebhookCallLog
{
    /**
     * Handle the event.
     *
     * @param  WebhookCallFailedEvent  $event
     * @return void
     */
    public function handle(WebhookCallFailedEvent $event)
    {
        $response = $event->response->getBody()->getContents();
        logger($response);
    }
}
```
But if I implement the `ShouldQueue` interface the response content is getting empty. Did anyone get into this trouble? Do you have anyway to solved this?

Thanks!

**Comment by jonaswebdev** [comment]
I'm facing the same issue on Laravel 11 with Laravel-webhook-server 3.8, even I don't use ShouldQueue.

**Reply by siarheipashkevich** [reply]
Maybe `properties` from these events are not serialized? For example: `GuzzleHttp\Psr7\Response` = `response` property
